### ISSUE TITLE
Not possible to install on 8.2 core master

### ISSUE THREAD

**Issue by jancborchardt** [raduvatav/polls#69]
I get this error on the app installation page:
```
An exception occurred while executing 'CREATE TABLE "oc_polls_events"
("id" VARCHAR(32) NOT NULL PRIMARY KEY AUTOINCREMENT,
"type" VARCHAR(16) DEFAULT NULL, "title" VARCHAR(128) NOT NULL,
"description" VARCHAR(1024) NOT NULL, "owner" VARCHAR(64) NOT NULL,
"created" VARCHAR(32) DEFAULT NULL, "access" VARCHAR(1024) DEFAULT NULL,
"expire" VARCHAR(32) DEFAULT NULL)':

SQLSTATE[HY000]: General error:
1 AUTOINCREMENT is only allowed on an INTEGER PRIMARY KEY
```

cc @v1r0x @raduvatav

**Comment by v1r0x** [comment]
Since I have no pc until the end of the month, my guess is that there is something wrong in the db xml. Or is it working on 8.1?

**Comment by RealRancor** [comment]
Fix is in #75 

The problem here is that the primary key ``id`` was changed from an ``integer`` to ``text`` in https://github.com/raduvatav/polls/commit/413c662ab2e1b4886703db2ba86d0a15de1b7eac but autoincrement is not allowed for text.

Maybe in further versions ``id`` could be an integer with autoincrement again and the "real" Poll ID could be moved to an own column. But for now this should be fine.
